lib/query_hook.rb

```diff
@@ -31,14 +32,11 @@ def command_line(filename)
     "ruby #{filename}"
   end
 
-  def post_process_file(_file, result, status)
-    if status == :failed && runtime_error_output?(result)
-      [sanitize_runtime_error_output(result), status]
-    elsif status == :failed && syntax_error_output?(result)
-      [sanitize_syntax_error_output(result), :errored]
-    else
-      super
-    end
+  def error_patterns
+    [
+      Mumukit::ErrorPattern.new(runtime_error_regexp),
```

Comment:
Shouldn't we use Failed instead of a plain ErrorPattern?

Reply:
👍